pkg/cassandra/nodetool/nodetool.go

```diff
@@ -0,0 +1,170 @@
+package nodetool
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/pborman/uuid"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/jetstack/navigator/pkg/cassandra/nodetool/client"
+)
+
+type NodeState string
+
+const (
+	NodeStateUnknown NodeState = "Unknown"
+	NodeStateNormal  NodeState = "Normal"
+	NodeStateLeaving NodeState = "Leaving"
+	NodeStateJoining NodeState = "Joining"
+	NodeStateMoving  NodeState = "Moving"
+)
+
+type NodeStatus string
+
+const (
+	NodeStatusUnknown NodeStatus = "Unknown"
+	NodeStatusUp      NodeStatus = "Up"
+	NodeStatusDown    NodeStatus = "Down"
+)
+
+type Node struct {
+	Host   string
+	ID     uuid.UUID
+	State  NodeState
+	Status NodeStatus
```

Comment:
What's the difference between State and Status? I see they have different values (above), but not sure what each of them mean. A brief comment here would help.

Reply:
I've added comments in the source code.